indico/htdocs/js/indico/jquery/ajaxdialog.js

```diff
@@ -81,6 +82,10 @@
         loadDialog();
 
         function loadDialog() {
+            if (options.content) {
+                showDialog({js: '', html: options.content[0].outerHTML});
```

Comment:
why not just use `options.content` and expect a HTML string (so you can pass `content: $(..).html()`)?